doc/cephfs/quota.rst

```diff
@@ -41,6 +41,13 @@ Limitations
    the directory the client is restricted too (e.g., ``/home/user``)
    or something nested beneath it.
 
+   In case of a kernel client, it needs to have access to the parent
+   of the directory inode on which quotas are configured in order to
+   enforce them. If quota is configured on a directory path
+   (e.g., ``/home/volumes/group``), the kclient needs to have access
+   to the parent (e.g., ``/home/volumes``).
+   See also: https://tracker.ceph.com/issues/55090
```

Comment:
Should we include an example command on how to create the user?

Reply:
We can do that.

Reply:
I will add the following example command.
```
ceph auth get-or-create client.guest1 mds 'allow r path=/home/volumes, allow rw path=/home/volumes/group' mgr 'allow rw' osd 'allow rw tag cephfs metadata=*' mon 'allow r'
```

Reply:
@vshankar Done